feat: documented invokable scripts within tasks functionality in the CLI (#4276)
influxdata/docs-v2: content/influxdb/v2.3/reference/cli/influx/task/create.md

|:-----|:------------------|:----------------------------------------------------------------------|:----------:|:----------------------|
| `-c` | `--active-config` | CLI configuration to use for command                                  | string     |                       |
|      | `--configs-path`  | Path to `influx` CLI configurations (default `~/.influxdbv2/configs`) | string     | `INFLUX_CONFIGS_PATH` |

> | `-r` | `--cron`          | [Cloud only] Cron expression to define when the task should run       | string     |                       |

Reviewer: ```suggestion
| `-r` | `--cron`          | _(InfluxDB Cloud only)_ Cron expression to define when the task runs             | string     |                       |
```

Reviewer: Is this really cloud only or will this be supported in InfluxDB 2.4?

> | `-e` | `--every`         | [Cloud only] Interval at which the task runs                          | string     |                       |

Reviewer: ```suggestion
| `-e` | `--every`         | _(InfluxDB Cloud only)_ Interval at which the task runs                          | string     |                       |
```

| `-f` | `--file`          | Path to Flux script file                                              | string     |                       |
| `-h` | `--help`          | Help for the `create` command                                         |            |                       |
|      | `--hide-headers`  | Hide table headers (default `false`)                                  |            | `INFLUX_HIDE_HEADERS` |
|      | `--host`          | HTTP address of InfluxDB (default `http://localhost:8086`)            | string     | `INFLUX_HOST`         |
|      | `--http-debug`    | Inspect communication with InfluxDB servers.                          | string     |                       |
|      | `--json`          | Output data as JSON (default `false`)                                 |            | `INFLUX_OUTPUT_JSON`  |

> | `-n` | `--name`          | [Cloud only] Name of the task                                         | string     |                       |

Reviewer: ```suggestion
| `-n` | `--name`          | _(InfluxDB Cloud only)_ Task name                                         | string     |                       |
```

| `-o` | `--org`           | Organization name (mutually exclusive with `--org-id`)                | string     | `INFLUX_ORG`          |
|      | `--org-id`        | Organization ID (mutually exclusive with `--org`)                     | string     | `INFLUX_ORG_ID`       |
|      | `--skip-verify`   | Skip TLS certificate verification                                     |            | `INFLUX_SKIP_VERIFY`  |

> |      | `--script-id`     | [Cloud only] Script ID that gets executed instead of Flux             | string     |                       |
> |      | `--script-params` | [Cloud only] JSON parameters for the script to be executed            | string     |                       |

Reviewer: ```suggestion
|      | `--script-id`     | _(InfluxDB Cloud only)_ Invokable script ID to execute             | string     |                       |
|      | `--script-params` | _(InfluxDB Cloud only)_ Invokable script JSON parameters            | string     |                       |
```

```sh
influx task create --file /path/to/example-task.flux
```

##### Create a task with a script
```sh
influx task create --name "my task" --every "10h" --script-id 0001234

> ```

Reviewer: ~~~suggestion
##### Create a task using an invokable script
```sh
influx task create \
  --name "my task" \
  --every "10h" \
  --script-id 0001234
```
~~~
